docker-compose.yml

```diff
@@ -1,18 +1,38 @@
-version: '2'
 services:
   erddap:
-    container_name: "erddap_gold_standard"
     image: axiom/docker-erddap:2.23-jdk17-openjdk
     restart: unless-stopped
     ports:
       - "8080:8080"
+    volumes:
+      - ./erddap/conf/robots.txt:/usr/local/tomcat/webapps/ROOT/robots.txt
+      - ./erddap/content:/usr/local/tomcat/content/erddap
+      - ./erddap/data:/erddapData
+      - ./datasets:/datasets
     environment:
+      #note: these can also be set in /usr/local/tomcat/bin/config.sh
```

Comment:
The documentation should be updated to reflect this change.
https://github.com/ioos/erddap-gold-standard/blob/gh-pages/_docs/customize-erddap.md#update-configsh